src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25912.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using OpenQA.Selenium;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25912 : _IssuesUITest
+	{
+		public Issue25912(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "ToolbarItem color when used with IconImageSource is always white";
+
+		[Test]
+		[Category(UITestCategories.ToolbarItem)]
+		public void VerifyToolbarItemIconColor()
+		{
+			App.WaitForElement("Label");
+			VerifyScreenshot();
```

Comment:
Snapshots available in the latest build:
![image](https://github.com/user-attachments/assets/ca930fe8-c987-44ce-9f24-245f8193a7f6)

Could you commit the images?